_includes/demo.html

```diff
@@ -2,7 +2,7 @@
   <div class="grid-container grid-container-desktop">
     <div class="grid-row grid-gap-2">
       <div class="grid-col-12">
-        <p><strong>This is a preview of the <em>new</em> Guide to the PRA.</strong> <a href="{{ '/about#feedback' | relative_url }}"><span>We're looking for your feedback</span> »</a></p>
+        <p><strong>This is a preview of the <em>new</em> Guide to the PRA.</strong> <a href="mailto:PRA@omb.eop.gov?subject=Feedback%20on%20PRA"><span>Email us your feedback</span> »</a></p>
```

Comment:
I included a way to obfuscate the email address to help reduce spam: https://github.com/GSA/digitalgov-pra/wiki/Adding-a-link-to-the-site-feedback-email-address

Reply:
Oh nice! thank you @hursey013 🎉

Reply:
Could I just suggest changing "Feedback on PRA" to "Feedback on the PRA Guide"?